src/bin/text/dune

```diff
@@ -1,9 +1,18 @@
 (documentation
   (package alt-ergo))
 
+(executable
+  (name gen_link_flags)
+  (libraries unix fmt stdcompat cmdliner)
+  (modules gen_link_flags)
+  (promote (until-clean)))
```

Comment:
There should be no need to promote this executable, let's not clutter the source directory.